Source/Wikitivity.Agent/WikitivityAgentHelper.cs

```diff
@@ -5,43 +5,156 @@
 using kCura.Relativity.ImportAPI;
 using Newtonsoft.Json;
 using Relativity.API;
+using Relativity.Services.Objects;
+using Relativity.Services.Objects.DataContracts;
 using System;
 using System.Collections.Generic;
 using System.Data;
 using System.Net;
+using System.Threading.Tasks;
+using ObjectType = kCura.Relativity.Client.DTOs.ObjectType;
 
 namespace Wikitivity.Agent
 {
 	class WikitivityAgentHelper
 	{
 		public static string IAPIMessages = "";
-
-		public bool DeleteUploadedRequestsFromRelativity(IRSAPIClient proxy, int workspaceID, List<WikitivityUploadsAgent.SingleRequestObject> listOfCompletedUploads)
+		public static Guid WikitivityRDOGuid = new Guid("C6196733-E2A6-48F4-9443-37990972EBA3");
+		
+		public bool DeleteUploadedRequestsFromRelativityOM(IObjectManager proxy, int workspaceID, List<WikitivityUploadsAgent.SingleRequestObject> listOfCompletedUploads)
 		{
-			proxy.APIOptions.WorkspaceID = workspaceID;
-
 			bool success = false;
 			try
 			{
-				List<Document> DocsToDelete = new List<Document>();
+				List<RelativityObjectRef> listofObjectstoDelete = new List<RelativityObjectRef>();
 				foreach (var singleDoc in listOfCompletedUploads)
 				{
-					DocsToDelete.Add(new Document(singleDoc.ArtifactID));
+					listofObjectstoDelete.Add(new RelativityObjectRef() { ArtifactID = singleDoc.ArtifactID });
 				}
 
-				var DeleteResultSet = proxy.Repositories.Document.Delete(DocsToDelete);
-				if (DeleteResultSet.Success)
+				var deleteRequest = new MassDeleteByObjectIdentifiersRequest();

```

Comment:
I haven't used ObjectManager code that much so I am not sure if all of the OM code used is per our recommendations. I would leave that up to the Platform folks to review it.